src/pages/login.svelte

```diff
@@ -12,6 +16,44 @@
     import MediaQuery from '~/components/utils/media-query.svelte'
     import Features from '~/components/elements/features.svelte'
     import UnicoveAnimated from '~/components/elements/unicove-animated.svelte'
+
+    const whalesplainerUrl = import.meta.env.SNOWPACK_PUBLIC_WHALESPLAINER_URL
+
+    let creatingAccount = false
+
+    async function createAccount(event: any) {
```

Comment:
Do you know what type I can use here? Couldn't figure it out so I ended up casting to any.

Reply:
`MouseEvent`

Reply:
I tried that, but then I get the following error: ```Error: Argument of type '(event: MouseEvent) => Promise<void>' is not assignable to parameter of type '(e: CustomEvent<Event>) => any'.
  Types of parameters 'event' and 'e' are incompatible.```

Reply:
Try `Event` maybe MouseEvent doesn't inherit from that for some reason

Reply:
yep! That worked!